app/assets/stylesheets/application.scss

```diff
@@ -18,6 +18,12 @@
 $borderbrown: #a52a2a;
 $backgroundorange: #ffb347;
 $fontgreen: #008000;
+.side-nav {
+  height: 100vh;
+}
+.body{
+  background-color: yellow;
```

Comment:
Color `yellow` should be written in hexadecimal form as `#ffff00`<br>Color literals like `yellow` should only be used in variable declarations; they should be referred to via variable everywhere else.